CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Changes
 
 - Upgraded Typescript dependency to 5.x
 - Replaced Karma test runner with Cypress
+- Added `@types/events` as a dependency. This resolves issues regarding missing Intellisense hinting for SDK objects that extend EventEmitters, as well as build issues in Angular webapps.
```

Comment:
It doesn't happen before 2.14 right? So it doesn't make sense to add this changelog.

Reply:
The missing Intellisense affects versions prior to `2.14.0-rc1`, so I will leave that tidbit in and remove the Angular thing.